new-dual-repo.sh

```diff
@@ -288,24 +295,21 @@ git commit -m "Added bare README"	|| die "git commit failed"
 
 if [[ ! -e .gitignore ]]
 then
-	# create a sane .gitignore for Mac users
-
-cat <<EOINPUT > .gitignore
-.DS_Store
-.DS_Store?
-.Spotlight-V100
-.Trashes
-Icon
-Icon*
-EOINPUT
-
+	# create a .gitignore
+	if [[ -e "$ROOT_DIR/config/gitignore" ]]
+	then
+		gitignore=$(<"$ROOT_DIR/config/gitignore")
+		echo "$gitignore" > .gitignore
+	else
+		echo "# ignore these files (1 file or pattern per line)\n" > .gitignore
+	fi
 fi
 
 [[ -s .gitignore ]] || die "Failed to create .gitignore"
 
 git add .gitignore || die "Failed to 'git add .gitignore'"
 
-git commit -m "Added .gitignore" || die "Failed to commit .gitignore"
+git commit -m "Adds .gitignore" || die "Failed to commit .gitignore"
```

Comment:
This was drummed into me at a couple of different jobs - The commit comments describe what the commit does to the codebase, not what you did to the codebase that's included in the commit. It's a matter of POV and tense. Very nitpicky.